### DISCUSSION TITLE
svgRoundQuietzone not working as expected

### DISCUSSION THREAD

**Discussion by Quy** [chillerlan/php-qrcode#241] [Q&A] [answered]
https://github.com/chillerlan/php-qrcode/blob/main/examples/svgRoundQuietzone.php

Using v5.0.1, the example is producing the QR without the GitHub logo and circle.
![qr](https://github.com/chillerlan/php-qrcode/assets/368084/16f017e4-e30b-44f2-9932-1b1916ca8a00)

Here is the HTML markup:
`<img src="svgRoundQuietzone.php" width="400" height="400">`

What should be the expected output? Is additional adjustments required? Thank you!

**Comment by codemasher** [comment, marked as answer]
Hi, you're using the example from `dev-main` instead of [the one from the `v5.0.x` branch](https://github.com/chillerlan/php-qrcode/tree/v5.0.x) - the difference is that in the main branch the option `QROptions::$outputType` doesn't exist anymore, which causes the output module not to be loaded. See also #223.

**Comment by Quy** [comment]
That did the trick. Thank you!!!
